client/src/featureform/metadata_repository_test.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+
+from client.src.featureform.metadata_repository import MetadataRepositorySQLite
+from client.src.featureform.resources import FeatureVariant, Feature, LabelVariant, Label, SourceVariant, \
```

Comment:
can we modify this import please?
```
from client.src.featureform.resources import (
      FeatureVariant, 
      Feature, 
      LabelVariant,
      ...
)
```
Also, order it too please. we can use alphabetical then by length of the word.